### ISSUE TITLE
Publish to openvsx

### ISSUE THREAD

**Issue by chlbri** [alexkrechik/VSCucumberAutoComplete#399]
I got gitpod.io subscription, but I cannot use your vscode extension. Please can you publish it to [openVSX](https://open-vsx.org/)

**Comment by alexkrechik** [comment]
Yes, I'll work on this during the nearest extension iteration. Thank you for the suggestion.
